Neos.Neos/Resources/Private/Styles/Foundation/_mixins.scss

```diff
@@ -433,27 +408,25 @@
   background-image: -webkit-gradient(linear, 0 0, 100% 0, from($startColor), to($endColor)); // Safari 4+, Chrome 2+
   background-image: -webkit-linear-gradient(left, $startColor, $endColor); // Safari 5.1+, Chrome 10+
```

Comment:
Probably all those vendor specifics can be removed too. Don't we have the autoprefixer running anyway?

Reply:
Right now we dont use autoprefixer in the none react ui... so this is a missing feature.
Maybe I can create a new PR as the webpack is now merged and add this.